.github/workflows/linting.yml

```diff
@@ -19,23 +19,16 @@ on:
 
 jobs:
   test_code:
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     strategy:
       max-parallel: 12
-      matrix:
-        python-version: ["3.10"]
-        os: [ubuntu-latest]
-
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v4
+      - uses: conda-incubator/setup-miniconda@v2
         with:
-          python-version: ${{ matrix.python-version }}
-          cache: "pip"
-      - name: Install dependencies
-        run: |
-          pip install flake8 pytest pytest_regressions
+          auto-activate-base: true
+          activate-environment: ""
```

Comment:
shoudl we choose a descriptive name here? (or remove it to use the default)

Reply:
@proppy removed. No longer needed.